vault/logical_system.go

```diff
@@ -1098,12 +1102,20 @@ func (b *SystemBackend) handlePluginCatalogUpdate(ctx context.Context, req *logi
 		return logical.ErrorResponse("missing command value"), nil
 	}
 
+	args := d.Get("args").([]string)
+	// For backwards compatibility, also accept args as part of command.
+	parts := strings.Split(command, " ")
+	if len(parts) <= 0 {
+		return logical.ErrorResponse("missing command value"), nil
+	}
+	args = append(parts[1:], args...)
```

Comment:
If they specify both `args` and put args in the command they'll end up with doubled args here. I think a better check would be to ensure that `len(parts) == 1` and if `args` is set and the length of `parts` is not 1, throw an error asking them to pick one or the other.